packages/flutter_tools/lib/src/macos/xcode.dart

```diff
@@ -78,12 +103,7 @@ class Xcode {
     return _xcodeSelectPath;
   }
 
-  bool get isInstalled {
-    if (xcodeSelectPath == null || xcodeSelectPath.isEmpty) {
-      return false;
-    }
-    return _xcodeProjectInterpreter.isInstalled;
-  }
+  bool get isInstalled => _xcodeProjectInterpreter.isInstalled;
```

Comment:
This is the only real behavior change.  The `xcodeSelectPath` check isn't necessary since `XcodeProjectInterpreter.isInstalled` would also be false if Xcode isn't installed.